launchable/utils/ingester_jar.py

```diff
@@ -0,0 +1,3 @@
+ingester_jar_version = "0.1.0"
```

Comment:
If `exe_deploy.jar` is updated, you should bump up this version number. I assume the number will be the same as the CLI version number.